app/controllers/registrations/user_profile_controller.rb

```diff
@@ -1,36 +0,0 @@
-# frozen_string_literal: true
-
-class Registrations::UserProfileController < Registrations::SchoolProfileController
-  before_action :load_school
-  before_action :check_school_available, only: :create
-
-  def new
-    @user = User.new
-  end
-
-  def create
-    @user = User.new(user_params)
-
-    ActiveRecord::Base.transaction do
-      @user.save!
-      @school.induction_coordinator_profiles.destroy_all
-      InductionCoordinatorProfile.create!(user: @user, schools: [@school])
-    end
-
-    session.delete(:school_id)
-    redirect_to :registrations_verification_sent
```

Comment:
Do we have some cypress helpers / mailer stuff that needs to go too?

And maybe some templates in notify?

Reply:
Good shout

Reply:
That's a big change, I'll remove in another PR